packages/instrumentation-typeorm/test/EntityManager.spec.ts

```diff
@@ -108,10 +112,12 @@ describe('EntityManager', () => {
 
         it('appends matching connection details to span', async () => {
             const [sqlite1, sqlite2] = await typeorm.createConnections([defaultOptions, options2]);
+            const manager1 = sqlite1.createEntityManager();
+            const manager2 = sqlite2.createEntityManager();
 
             const user = new User(1, 'aspecto', 'io');
-            await sqlite1.manager.save(user);
```

Comment:
do we want to keep a test with the old syntax as well? just to make sure it works both ways

Reply:
good idea